migration/idempotent/008-tracing-functions.sql

```diff
@@ -343,6 +343,7 @@ BEGIN
     SELECT * INTO _tag
     FROM _ps_trace.tag
     WHERE key = _key
+    AND pg_catalog.sha512(value::TEXT::BYTEA) = pg_catalog.sha512(_value::TEXT::BYTEA)
```

Comment:
We have to put `pg_catalog.sha512(_value::TEXT::BYTEA)` on the right hand side in three places to get the index scan, right? That's why we cannot calc the sha once into a variable and reuse it?